tests/test_units/test_rms_context.py

```diff
@@ -295,6 +296,139 @@ def test_cube_export_file_set_name(rmssetup, rmsglobalconfig, cube):
     )
 
 
+@inside_rms
+def test_cube_export_file_set_name_as_observation(rmssetup, rmsglobalconfig, cube):
+    """Export the cube to file with correct metadata and name, is_observation."""
+    logger.info("Active folder is %s", rmssetup)
+    os.chdir(rmssetup)
+
+    edata = dataio.ExportData(config=rmsglobalconfig)  # read from global config
+
+    output = edata.export(cube, name="MyCube", is_observation=True)
+    logger.info("Output is %s", output)
+
+    assert str(output) == str(
+        (edata._rootpath / "share/observations/cubes/mycube.segy").resolve()
+    )
+
+
+@inside_rms
+def test_cube_export_file_set_name_as_observation_forcefolder(
+    rmssetup, rmsglobalconfig, cube
+):
+    """Export the cube to file with correct metadata and name, is_observation.
+
+    In addition, use forcefolder to apply "seismic" instead of cube
+    """
+    logger.info("Active folder is %s", rmssetup)
+    os.chdir(rmssetup)
+
+    edata = dataio.ExportData(config=rmsglobalconfig)  # read from global config
+
+    # use forcefolder to apply share/observations/seismic
+    output = edata.export(
+        cube,
+        name="MyCube",
+        fmu_context="realization",
+        is_observation=True,
+        forcefolder="seismic",
+    )
+    logger.info("Output after force is %s", output)
+
+    assert str(output) == str(
+        (edata._rootpath / "share/observations/seismic/mycube.segy").resolve()
+    )
+
+
+@inside_rms
+def test_cube_export_as_observation_forcefolder_w_custom_subfolder(
```

Comment:
Perhaps rename to `....forcefolder_as_path` or something to avoid confusing with the `subfolder` argument?